playwright.config.ts

```diff
@@ -36,8 +36,8 @@ const config: PlaywrightTestConfig = {
     /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
     actionTimeout: 0,
     /* Base URL to use in actions like `await page.goto('/')`. */
-    // baseURL: 'http://localhost:3000',
-    baseURL: "https://docs.camunda.io",
+    baseURL: "http://localhost:3000",
```

Comment:
Did you mean to leave this as localhost?

Reply:
Yes! These tests are never run by CI, and they're rarely even run locally. I think the local use case makes more sense though, regardless.